components/tools/OmeroPy/test/unit/clitest/test_admin.py

```diff
@@ -332,3 +332,50 @@ def testExplicitPorts(self, registry, ssl, tcp, webserver, prefix):
         self.cli.invoke(self.args, strict=True)
 
         self.check_ice_config(**kwargs)
+
+
+class TestAdminJvmCfg(object):
+
+    @pytest.fixture(autouse=True)
+    def setup_method(self, monkeypatch):
```

Comment:
Ah, I see what you mean. This seems to be mixing the two idioms of `setup_method` and `fixture`. If `setup_method` as it is doesn't work for this use case then maybe a different name for the fixture method would avoid any confusion?